Makefile

```diff
@@ -47,23 +61,23 @@ check: check-wasm $(RUNTIMES:%=check-%);
 
 check-common: check-wasm;
 check-wasm:
-	$(CARGO) +nightly fmt $(TARGET_FLAG) -p oci-tar-builder -p wasi-demo-app -p containerd-shim-wasm -p containerd-shim-wasm-test-modules -- --check
+	CARGO= $(CARGO) +nightly fmt -p oci-tar-builder -p wasi-demo-app -p containerd-shim-wasm -p containerd-shim-wasm-test-modules -- --check
```

Comment:
why is CARGO being set here?

Reply:
It looks like `rust-fmt` uses the `CARGO` env-var for something, and it fails when I set to `cross`.

Reply:
maybe a note here to call that out?